src/components/views/elements/Field.tsx

```diff
@@ -195,12 +196,18 @@ export default class Field extends React.PureComponent<PropShapes, IState> {
         this.props.onBlur?.(ev);
     };
 
+    // Memoize latest result as some validation functions can be costly, e.g. API hits
+    private onValidate = memoizeOne(
```

Comment:
Why use `memoizeOne` over `memoize` in lodash?
I understand they work slightly differently, but i'm wondering what is the rationale behind only memoizing the last validation call? 

Is it to save memory? Or are there cases where the validation functions aren't pure? 

My preference would be to get rid of `memoize-one` unless there's a compelling reason to keep it.